log/syslog_formatter.go

```diff
@@ -0,0 +1,84 @@
+package log
+
+import (
+	"fmt"
+	"log/syslog"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var ceeTag = []byte("@cee:")
+
+var _ logrus.Formatter = &syslogger{}
```

Comment:
not needed removed, this was copied from our internal logging library that also uses logrus